plugins/http_plugin/http_plugin.cpp

```diff
@@ -144,6 +159,12 @@ namespace eos {
                      if (!my->access_control_allow_origin.empty()) {
                         con->append_header("Access-Control-Allow-Origin", my->access_control_allow_origin);
                      }
+                     if (!my->access_control_allow_headers.empty()) {
```

Comment:
While it does no harm to include `Access-Control-Allow-Headers` on every response, the standards are such that it is  only meaningful as part of an `OPTIONS` request. 

https://www.w3.org/TR/cors/#resource-requests (section 6.2.4)

this does not apply to `Access-Control-Allow-Origin` and `Access-Control-Allow-Credentials` as those are included on preflighted, simple CORS, regular and redirect responses.

Reply:
perhaps conditionally include the `OPTIONS` headers based on the method?

for reference, there are other headers we may want to consider that are `OPTIONS` only:

- `Access-Control-Request-Method` to limit to `POST` `GET` and `OPTIONS`?
- `Access-Control-Max-Age` to some resonable limit to reduce future preflights from the same client?

Reply:
I believe the web wallet team is going to propose some changes to the http interface. These sort of items would likely be part of that. These were added as the minimum needed for the web wallet demo.